src/extension/background-script/connectors/lnbits.ts

```diff
@@ -96,7 +96,7 @@ class LnBits implements Connector {
         data: {
           checking_id: string;
           pending: boolean;
-          amount: string;
+          amount: number;
```

Comment:
It's actually a number:
```
  /*
  LNBits Swagger Docs: https://legend.lnbits.org/devs/swagger.html#/default/api_payments_api_v1_payments_get
  Sample Response:
    [
      {
        "checking_id": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
        "pending": false,
        "amount": 2000,
```

Reply:
![image](https://user-images.githubusercontent.com/1016218/196693917-548aba59-2aa4-4c1a-9529-9263bef615fb.png)